icu4j/main/classes/core/src/com/ibm/icu/text/PersonName.java

```diff
@@ -92,17 +101,21 @@ private NameField(String name) {
 
         /**
          * Returns the NameField's display name.
-         * @internal
+         * @internal ICU 72 technology preview
```

Comment:
@richgillam @macchiati Should the PersonName* APIs that only have `@internal` remain like that (i.e., never intended to become public), or should they be "tech preview" like Yoshito is tagging them here?

Reply:
If I had something as just "@internal" without the "tech preview" verbiage, it's because I intended it to really be internal (i.e., never to be part of the public API), not just internal because it was part of a tech preview.  In other words, don't change this.

Reply:
@richgillam This specific one is for toString(). I think it's not a good idea to make toString() @internal if a class becomes a public ICU API.